docs/device_automation_trigger.md

```diff
@@ -66,7 +66,7 @@ from homeassistant.const import (
 async def async_get_triggers(hass, device_id):
     """Return a list of triggers."""
 
-    device_registry = await hass.helpers.device_registry.async_get_registry()
+    device_registry = hass.helpers.device_registry.async_get()
```

Comment:
```suggestion
    device_registry = async_get(hass)
```

Reply:
should I add the import too, or can we assume `dr` is known?

Reply:
Sure, that will not harm :)